Godeps/Godeps.json

```diff
@@ -907,152 +907,152 @@
 		},
 		{
 			"ImportPath": "github.com/docker/distribution/digestset",
-			"Comment": "v2.6.0-rc.1-209-gedc3ab29",
+			"Comment": "v2.6.0-rc.1-209-gedc3ab2",
 			"Rev": "edc3ab29cdff8694dd6feb85cfeb4b5f1b38ed9c"
 		},
 		{
 			"ImportPath": "github.com/docker/distribution/reference",
-			"Comment": "v2.6.0-rc.1-209-gedc3ab29",
+			"Comment": "v2.6.0-rc.1-209-gedc3ab2",
 			"Rev": "edc3ab29cdff8694dd6feb85cfeb4b5f1b38ed9c"
 		},
 		{
 			"ImportPath": "github.com/docker/docker/api",
-			"Comment": "docs-v1.12.0-rc4-2016-07-15-7401-g4f3616fb1",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-7401-g4f3616f",
```

Comment:
we can seee the Godeps.json file in openshift. Its Comment is also `docs-v1.12.0-rc4-2016-07-15-7401-g4f3616f`.
https://github.com/openshift/origin/blob/master/Godeps/Godeps.json#L1219
```
		{
			"ImportPath": "github.com/docker/docker/api",
			"Comment": "docs-v1.12.0-rc4-2016-07-15-7401-g4f3616f",
			"Rev": "4f3616fb1c112e206b88cb7a9922bf49067a7756"
		},
```